test/controllers/api/docs_controller_test.exs

```diff
@@ -32,83 +27,62 @@ defmodule HexWeb.API.DocsControllerTest do
 
     conn = get build_conn(), "docs/phoenix/index.html"
     assert conn.status == 200
-    assert conn.resp_body == "HEYO"
+    assert conn.resp_body == "phoenix v0.0.1"
```

Comment:
If I am only testing status I usually use `assert conn.status == xxx` but if I am testing both status and body I usually use `assert reponse(conn, xxx) == "body"`.